torch/distributed/_sharded_tensor/__init__.py

```diff
@@ -225,7 +231,8 @@ def full(sharding_spec: ShardingSpec,
          process_group=None,
          init_rrefs=False):
     """
-    Creates a full :class:`ShardedTensor`. Needs to be called on all ranks in an SPMD fashion.
+    Creates a :class:`ShardedTensor` filled with fill_value. The tensor’s dtype
+        is inferred from fill_value. Needs to be called on all ranks in an SPMD fashion.
```

Comment:
We should clarify the behavior clearly here that if dtype is specified that overrides the type inferred from fill_value.

Reply:
updated